webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const nodeExternals = require('webpack-node-externals');
```

Comment:
Require is fine here, right?

Reply:
i think so as it's in webpack config itself